cluster-autoscaler/cloudprovider/azure/azure_scale_set_instance_cache_test.go

```diff
@@ -19,26 +19,12 @@ package azure
 import (
 	"fmt"
 	"testing"
-	"time"
 
 	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2022-08-01/compute"
 	"github.com/golang/mock/gomock"
```

Comment:
It gives the following error: `github.com/golang/mock/gomock" imported and not used`

so we have to remove the 	"github.com/golang/mock/gomock" from the import.

Reply:
@comtalyst, IMO the maintainers of this repo have the permission to update the PR. IIRC, @x13n updated one of this Kind of PR.
@x13n could you please take a look?

Reply:
Looks like I was able to add a commit, the code should build now.

Reply:
Thanks @x13n, for fixing it.